### PR TITLE
Add support to Mid 2019 Blade Advanced Mercury White

### PR DESCRIPTION
**Pull request by mgruppi**
#834

### CODE REVIEW

File: daemon/openrazer_daemon/hardware/keyboards.py [hunk 1]
```diff
@@ -1113,3 +1113,28 @@ class RazerBlade2019Adv(_RippleKeyboard):
...
+               'set_ripple_effect', 'set_ripple_effect_random_colour']
+
+    DEVICE_IMAGE = "https://assets2.razerzone.com/images/blade-15/shop/blade15-mercury-1.jpg"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": "https://assets.razerzone.com/eeimages/support/products/1482/blade15.png",
+        "side_img": "https://assets2.razerzone.com/images/blade-15/blade15-usp-ports-advanced-model.jpg",
+        "perspective_img": "https://d4kkpd69xt9l7.cloudfront.net/sys-master/root/h78/h94/9126618628126/razer-blade-15-gallery07.jpg"
+    }
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/keyboards.py]: Please remove this block

**Review by z3ntu — CHANGES_REQUESTED** [review]
Looks good overall, just a small comment.

Do all effects work with that PR?

File: daemon/openrazer_daemon/hardware/keyboards.py (hunk 1 above)
Comment (mgruppi) [reply daemon/openrazer_daemon/hardware/keyboards.py]: Yes, I've got a Blade Advanced Early 2019 Mercury White and all the effects work fine. The Mid 2019 model has just a different pid. 
I'll make the changes you requested soon enough.

Comment (mgruppi) [reply daemon/openrazer_daemon/hardware/keyboards.py]: Done.

**Review by z3ntu — APPROVED** [review]
LGTM, thanks!

File: daemon/openrazer_daemon/hardware/keyboards.py (hunk 1 above)
Comment (oplab-team) [reply daemon/openrazer_daemon/hardware/keyboards.py]: > Looks good overall, just a small comment.
> 
> Do all effects work with that PR?

I couldn't get my wave effect to work properly. It doesn't have that fade-in/fade-out effect. Also, I couldn't get my keyboard work with different colors for each key. It seems that all keys are stuck with the same color. No independent color changing.

**Comment by cesarvargas00** [comment]
> Looks good overall, just a small comment.
> 
> Do all effects work with that PR?

I couldn't get my wave effect to work properly. It doesn't have that fade-in/fade-out effect. Also, I couldn't get my keyboard work with different colors for each key. It seems that all keys are stuck with the same color. No independent color changing.

**Pull request merged**